packages/gatsby-theme-docs/src/components/theme-provider.js

```diff
@@ -4,9 +4,12 @@ import { useStaticQuery, graphql } from 'gatsby';
 import { Reset, Globals } from '@commercetools-docs/ui-kit';
 import { SiteDataContext } from '../hooks/use-site-data';
 import useAdditionalSiteData from '../overrides/use-additional-site-data';
+import CodeExampleLanguageContext from '../hooks/use-code-example-language';
 import ErrorBoundary from './error-boundary';
 
 const ThemeProvider = (props) => {
+  const [language, setLanguage] = React.useState('test');
```

Comment:
The name is too generic and it can be confused with "language/locale".
We should name it like `codeExampleLanguage`

Reply:
Also remember to remove `test`

```suggestion
  const [language, setLanguage] = React.useState('');
```

Reply:
We should probably make the default language configurable for each website, as a theme option.